services/statestorage/layered_state_test.go

```diff
@@ -0,0 +1,203 @@
+package statestorage
+
+import (
+	"fmt"
+	"github.com/orbs-network/go-mock"
+	"github.com/orbs-network/orbs-network-go/services/statestorage/adapter"
+	"github.com/orbs-network/orbs-spec/types/go/primitives"
+	"github.com/orbs-network/orbs-spec/types/go/protocol"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestWriteAtHeight(t *testing.T) {
+	persistenceMock := &StatePersistenceMock{}
+	persistenceMock.
+		When("Read", primitives.ContractName("c"), "k1").
+		Return((*protocol.StateRecord)(nil), false, nil).
+		Times(1)
+	d := newDriver(persistenceMock, 5)
+	d.write(1, "c", "k1", "v1")
+
+	v, exists, err := d.read(0, "c", "k1")
+	require.NoError(t, err)
+	require.EqualValues(t, false, exists)
+
+	v, exists, err = d.read(1, "c", "k1")
+	require.NoError(t, err)
+	require.EqualValues(t, true, exists)
+	require.EqualValues(t, "v1", v)
+
+	v, exists, err = d.read(200, "c", "k1")
+	require.Error(t, err)
+
+	ok, errCalled := persistenceMock.Verify()
```

Comment:
no need to assert both boolean and error result of `Verify`, just assert no error.

Reply:
ignoring ok everywhere. but if it's never important why is it included.. ?